astacus/common/access_log.py

```diff
@@ -0,0 +1,16 @@
+"""
+Copyright (c) 2024 Aiven Ltd
+See LICENSE for details
+"""
+from collections.abc import Sequence
+
+import logging
+
+
+class AccessLogLevelFilter(logging.Filter):
+    def filter(self, record: logging.LogRecord) -> bool:
+        # Lower the log level of GET requests from INFO to DEBUG
+        if isinstance(record.args, Sequence) and len(record.args) >= 2 and record.args[1] == "GET":
```

Comment:
do we need to do all this work at each request? Isn't it possible (or desiderable) to do this at astacus level with a logger.debug vs logger.info call in the right place?